### PR TITLE
Edit LONGPOWER contract address

### PR DESCRIPTION
**Pull request by psyduckasa**

### CODE REVIEW

**Review by e-asia — APPROVED** [review]

File: spaces/long/index.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "symbol": "LONGPOWER",
   "name": "Long",
   "defaultView": "core",
-  "address": "0xf3c13090b8aD08d8EeE1C3fd224E034f1F44820c",
-  "token": "0xf3c13090b8aD08d8EeE1C3fd224E034f1F44820c",
+  "address": "0x2ece5a5f2bf395e77d2f3e645b1611a0739b4ef9",
+  "token": "0x2ece5a5f2bf395e77d2f3e645b1611a0739b4ef9",
   "core": [
     "0x0AF72515C8E832f079A628e6744faae20846c6C3"
   ],
```
Comment (e-asia) [inline spaces/long/index.json:10]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/long/index.json (hunk 1 above)
Comment (psyduckasa) [reply spaces/long/index.json:12]: Done. Added chainId and decimals parameters. @e-asia

**Review by e-asia — APPROVED** [review]

File: spaces/long/index.json (hunk 1 above)
Comment (e-asia) [reply spaces/long/index.json:12]: @psyduckasa If you change the base token address your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

Comment (psyduckasa) [reply spaces/long/index.json:12]: @e-asia Yes, it's OK, new LONGPOWER contract fixes incorrect pool id.

**Comment by psyduckasa** [comment]
@e-asia Looks like the code has been manually merged in?
https://github.com/bonustrack/snapshot-spaces/blob/master/spaces/long/index.json

**Pull request merged**